api/seeds/shared/set-up-users.js

```diff
@@ -108,7 +108,7 @@ const createUsersToAdd = async (knex, oktaClient) => {
     oktaAffiliations.push({
       user_id: stateAdmin.id,
       state_id: 'ak',
-      role_id: stateAdminRoleId,
+      role_id: stateStaffRoleId,
```

Comment:
I want to make them a "match" for the admin certification process.  They shouldn't get to be an admin until they get matched up.  This may not be ideal long term, but for now it helps the testing process.

Reply:
Ok, true. I didn't think about that. That sounds good for now though.